ml_service/mocks/azureml/core/environment.py

```diff
@@ -0,0 +1,28 @@
+from azureml.core.environment import DockerSection, PythonSection
+
+
+class MockedEnvironment:
```

Comment:
This is the custom Mock used in one of the unit tests. It is irrelevant for the other two `automock` examples.